adapter/denyChecker/adapter_test.go

```diff
@@ -18,14 +18,8 @@ import (
 	"testing"
 
 	"istio.io/mixer/pkg/adapter"
-	"istio.io/mixer/pkg/adaptertesting"
 )
 
-func TestBuilderInvariants(t *testing.T) {
```

Comment:
Yes, I will add it back.
I wanted to get the structure in.
 
I also think that this is something we can do across the board, so every adapter need not implement it.